content/modeling/modeling_4.md

```diff
@@ -71,13 +71,15 @@ redshift_database=redshift_database
 redshift_port=redshift_port
 redshift_user=redshift_user
 redshift_password=redshift_password
+redshift_schema=redshift_schema
 ```
 
-Additionally, regardless of your data warehouse, you need to provide the conversion window start and end date and optionally the attribution model if you wish not to use the default, e.g.:
+Additionally, regardless of your data warehouse, you need to provide the conversion window start and end date, the warehouse you are using (one of snowflake, bigquery, databricks, or redshift) and optionally the attribution model if you wish not to use the default, e.g.:
```

Comment:
Being very nit picky, but we capitalise warehouses elsewhere in the accelerator
```suggestion
Additionally, regardless of your data warehouse, you need to provide the conversion window start and end date, the warehouse you are using (one of Snowflake, BigQuery, Databricks, or Redshift) and optionally the attribution model if you wish not to use the default, e.g.:
```

Reply:
I think it technically will work regardless, but in theory the value should be lowercase

Reply:
Ah I misread read that these were the warehouses available rather than the actual input values so ignore me 🤦  code-formatting them would be a nice touch!